ml-notebooks/image_classification/pcam/02_1_vertex_ai_pcam.ipynb

```diff
@@ -310,6 +310,41 @@
     "print(BUCKET)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Copy the Python package with the training code to your bucket."
```

Comment:
I think it would be worthwhile to add a bit of commentary on why these steps may be necessary. Something about why we would want to copy it from the workspace bucket to this bucket.

Reply:
Good point, I'll add a bit more explanation before merging.